apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/ViewPoolCandidatePage.tsx

```diff
@@ -65,6 +68,8 @@ const PoolCandidate_SnapshotQuery = graphql(/* GraphQL */ `
   query PoolCandidateSnapshot($poolCandidateId: UUID!) {
     poolCandidate(id: $poolCandidateId) {
       ...CandidateExpiryDateDialog
+      ...RemoveCandidateDialog
```

Comment:
Nothing to do with this PR, but adding other fragments to this query is goign to make it harder to complete #9499 someday.  Perhaps that snapshot part of it could be extracted to its own framgent sometime.